### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dougwettlaufer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
...
+
+Before building you will first need to set up your local m2. 
+
+First add this to ~/.m2/settings.xml
+
+```yaml
+<servers>
+    <server>
+        <id>gh</id>
+        <username>YOUR_USERNAME</username>
```
Comment (jorgebay) [inline README.md:13]: Usually public packages shouldn't require authentication. Is there a reason for this?

[Doc states that it only requires auth for publishing packages](https://docs.github.com/en/packages/using-github-packages-with-your-projects-ecosystem/configuring-apache-maven-for-use-with-github-packages#authenticating-to-github-packages) which makes sense to me.

Comment (dougwettlaufer) [reply README.md:13]: The docs are misleading https://stackoverflow.com/questions/58438367/how-to-access-maven-dependecy-from-github-package-registry-beta

Comment (dougwettlaufer) [reply README.md:13]: Anonymous access is currently a beta feature for GH Containers so hopefully they add it to Packages soon

**Comment by jorgebay** [comment]
Given that GH packages require auth, wouldn't it be a better experience for users if we deploy on maven central?

Maven users have access to maven repo by default.

**Review by dimas-b — APPROVED** [review]

**Review by EricBorczuk — APPROVED** [review]

**Pull request merged**